tests/conftest.py

```diff
@@ -38,6 +38,16 @@ def _factory(logger, level=logging.INFO):
     return _factory
 
 
+@pytest.fixture
```

Comment:
You don't need to define this fixture. The existing `caplog_factory` is changed to add the log level parameter. You can use it as follows in your test files:
```
    logger = logging.getLogger("aap_eda.services.activation.tee_system_logger")
    eda_log = caplog_debug_factory(logger, logging.DEBUG)
```